api/v1/cluster_webhook.go

```diff
@@ -213,33 +215,21 @@ func (r *Cluster) defaultInitDB() {
 
 // defaultRecovery enriches the recovery with defaults if not all the required arguments were passed
 func (r *Cluster) defaultRecovery() {
```

Comment:
The benifit of introducing in databaseInfo is when this value is not specified, which means user does not want to setup the application database after recovery.